azerothcore/azerothcore-wotlk
# fix(Core/Script): Startup error in spell_bwl_shadowflame

## Changes Proposed:
-  Validate used spells
-  Correct SPELL_EFFECT_* type in SpellEffectFn to match DBC data (The original issue)


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4412



## Tests Performed:
- Builds
- Tested in-game



## How to Test the Changes:

The absolute easiest is to get two chars, either opposite factions or into a duel.

1. .cast 22539
2. See that the player gets aura 22682 on them.
3. .aura 22683 (On the receiving player)
4. .cast 22539
5. See that the player does not get aura 22682 on them


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
